lesson16/test.json

```diff
@@ -0,0 +1,141 @@
+{
+  "data": [
+    {
+      "category": "news",
+      "id": "9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6a",
+      "isActive": true,
+      "image": {
+        "img": "/lesson16/img/news.png",
+        "alt": "newsの画像"
+      },
+      "article": {
+        "article1": {
+          "id": "9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb7a",
+          "title": "ニュース1タイトルニュース1タイトル",
+          "url": "news/article1.html",
+          "commentCount": 99,
```

Comment:
As Morita-sensei once commented in disacode, it is not very practical to set the comment as a numerical value.
The following is the JSON data I wrote.

``` json
 "comment": [
    {
      "id": "d70d67e9-fe9d-404e-907c-2dfc22582eb1",
      "name": "Aaron",
      "icon": "https://i.postimg.cc/V6vMMNnK/face-obaasan-trouble.jpg",
      "detail": "11月18日のニュースに関するコメント"
    },
    {
      "id": "d70d67e9-fe9d-404e-907c-2dfc22582eb3",
      "name": "Daniel",
      "icon": "https://i.postimg.cc/fRkhGnc6/face-girl.jpg",
      "detail": "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,"
    }
  ]

```


I don't think I need to modify it this time.
I'm glad if it helps you.

Reply:
So it is stored in JSON data in this format. I'm learning a lot!
I used Sae-san's JSON data as a reference and modified it.

Thanks for all the comments you've written so far! I've learned a lot from you!😊🙏